sympy/concrete/summations.py

```diff
@@ -421,7 +421,8 @@ def is_convergent(self):
         sym = self.limits[0][0]
         lower_limit = self.limits[0][1]
         upper_limit = self.limits[0][2]
-        sequence_term = self.function
+        sequence_term = simplify(self.function)
```

Comment:
What about `self.function.simplify()`? IMO, `simplify` is slow.

Reply:
Will make the necessary change...